nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
 
+  // Reset profiling timers and counters within event_delivery_manager
+  kernel().event_delivery_manager.reset_timers_counters();
```

Comment:
Not sure what you mean about the nestio backend. In my view it is not re-initialized with each `run`, otherwise we would still have the sionlib problems.

My opinion about the position of `reset_timers_counters` is not very strong. Thus, if you like it as it is, I have no further objections.